src/app/components/AssignmentTimeline/AssignmentTimelineCard.tsx

```diff
@@ -31,13 +31,13 @@ export default class AssignmentTimelineCard extends React.PureComponent<Assignme
 
 
         let backgroundColor = "";
-        if (assignmentType === ASSIGNMENT_TYPES.courtSecurity) {
+        if (WORK_SECTIONS[workSectionId] === WORK_SECTIONS.COURT) {
```

Comment:
I think we should eventually stick some utility methods inside... probably the API for getting colours, styles icons etc for specific types of things (like this).  That way we can ensure it's consistent across the board and it opens it up for custom colours per courthouse etc in the future.